buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -171,8 +173,8 @@ RUN cd /usr/src \
   && rm -rf /usr/src/git-2.5.3
 
 # Install ant version 1.10.5.
-RUN wget https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.zip \
-  && unzip apache-ant-1.10.5-bin.zip -d /opt \
+COPY apache-ant-1.10.5-bin.zip . 
```

Comment:
Why this change? `apache-ant-1.10.5-bin.zip` isn't (and shouldn't be) in the openj9 repo.

Reply:
The original link for downloading ant expires. Sorry, I am not sure why we use this version. @AdamBrousseau

Reply:
The original link expired. We updated it in https://github.com/eclipse/openj9/commit/e22aa4624adf6de88a0e92a1a2c80f28c71ad258#diff-98f2a595424a7af76747ea2c636f6d19. The `COPY` was put in temporarily because @jennychenyj's laptop wasn't resolving the url. This can be removed now.

Reply:
It was just removed. Sorry. I forgot we made this change to test the dockerfile. Thanks for explaining it! @AdamBrousseau